invenio_rdm_records/requests/user_moderation/actions.py

```diff
@@ -8,12 +8,48 @@
 
 from invenio_access.permissions import system_identity
 
-from invenio_rdm_records.proxies import current_rdm_records_service
+from ...proxies import current_rdm_records_service
 
 
 def on_block(user_id, uow=None, **kwargs):
     """Removes records that belong to a user."""
-    pass
+    user_id = str(user_id)
```

Comment:
could you add handling "on_restore" action as well ? I think it is also already possible from the record side

Reply:
sure, i was about to add it already – but then i wasn't sure about _which_ of the user's deleted records to restore.
all of them?

because some records may have been deleted already before the user was blocked, and i guess we don't want to restore them...

Reply:
thats a good question, I will raise it with Lars

Reply:
I would restore everything for now. Later we need to perhaps allow an admin to select what to restore. For now I would do everything. Usually a blocked user is a misbehaving  and will normally not have any prior deleted records.